.github/workflows/broken-links-crawler.yml

```diff
@@ -14,10 +14,17 @@ permissions:
   contents: write
 
 jobs:
+  debug-event:
```

Comment:
Do you want to leave it here, or was it just  used during some debugging ?

Reply:
I would like to leave it there for a while, to gather evidence, eventually remove.